src/controllers/trust.details.controller.ts

```diff
@@ -6,6 +6,6 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
   getTrustDetails(req, res, next, false, false);
 };
 
-export const post = (req: Request, res: Response, next: NextFunction) => {
-  postTrustDetails(req, res, next, false, false);
+export const post = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Identified that this was missing during testing. Function should be asynchronous and awaiting the response from `postTrustDetails`.